docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -910,10 +912,9 @@ generally has the form `[[SOURCE-VOLUME|HOST-DIR:]CONTAINER-DIR[:OPTIONS]]`.
 
 If `SOURCE-VOLUME` starts with `.`, Quadlet resolves the path relative to the location of the unit file.
 
-As a special case, if `SOURCE-VOLUME` ends with `.volume`, a Podman named volume called
-`systemd-$name` is used as the source, and the generated systemd service contains
-a dependency on the `$name-volume.service`. Such a volume can be automatically be lazily
-created by using a `$name.volume` Quadlet file.
+Special case:
+
+* If `SOURCE-VOLUME` ends with `.volume`, a Podman named volume called `systemd-$name` is used as the source, and the generated systemd service contains a dependency on the `$name-volume.service`. Such a volume can be automatically be lazily created by using a `$name.volume` Quadlet file.
```

Comment:
> Such a volume can be automatically be lazily created

I know this exists in the original doc, but it's incorrect. The `.volume` file must exist